spec/classes/puppet_profile_puppetdb_spec.rb

```diff
@@ -67,6 +67,7 @@
             :disable_ssl        => false,
             :listen_address     => '127.0.0.1',
             :ssl_listen_address => '0.0.0.0',
+            :manage_dbserver    => true,
```

Comment:
Adding that test surfaced a bug in the puppetlabs/postgresql module: https://tickets.puppetlabs.com/browse/MODULES-4054.